.github/workflows/continuous-integration.yml

```diff
@@ -107,9 +106,6 @@ jobs:
       - name: Setup Postgres
         uses: ./.github/actions/setup-postgres
 
-      - name: Setup Postgres wallet plugin
-        uses: ./.github/actions/setup-postgres-wallet-plugin
-
       - name: Setup NodeJS
         uses: actions/setup-node@v3
```

Comment:
We can update setup-onde action to v4, as suggested by our friend @dependabot  in #1624.

BTW do we still need to install libindy in CI?

Reply:
I've removed libindy, not going to update the setup-node now, as I don't want to mess with the ci more until the publishing works again